texascale-org/static/texascale-org/css/src/template.article.image-map.css

```diff
@@ -5,5 +5,5 @@
 
 /* COMPONENTS */
 
-@import url("../../../../../../taccsite_cms/static/site_cms/css/src/_imports/components/c-image-map.css");
+@import url("_imports/components/c-image-map.css");
 @import url("_imports/components/c-image-map.css");
```

Comment:
Crap. They are now. Thank you. Must fix.

<details><summary>This would break styles on two pages on Texascale.</summary>

- [TACC's Global Impact](https://texascale.org/2019/visualizing-science/taccs-global-impact/)
- [The Frontera Secret Sauce](https://texascale.org/2020/visualizing-science/frontera-secret-sauce/)

</details>

Reply:
Wes, check for other instances (unlikely but possible).

Reply:
Fixed via e085fec. (I did find other different kinds of paths to fix.)